t/food.t

```diff
@@ -72,6 +72,9 @@ is( normalize_serving_size("42\N{U+516C}\N{U+5347}"), 42000 );
 is( unit_to_g(42, "\N{U+516C}\N{U+5347}"), 42000 );
 is( g_to_unit(42000, "\N{U+516C}\N{U+5347}"), 42 );
 
+#test for a bug
```

Comment:
Maybe we could add the link to the actual bug.

Reply:
> 2 unrelated files were deleted in this PR.

@stephanegigandet This is strange. Not sure how that happened but I'll be more careful now. I don't remember staging these changes.

Reply:
Done and done